cli/src/main/java/com/devonfw/tools/ide/tool/intellij/Intellij.java

```diff
@@ -23,6 +36,61 @@ public Intellij(IdeContext context) {
     super(context, "intellij", Set.of(Tag.INTELLIJ));
   }
 
+  @Override
+  protected void runIde(String... args) {
+
+    install(true);
+
+    Step stepRun = this.context.newStep("Running IntelliJ");
+    try {
+      ProcessResult result;
+      if (this.context.getSystemInfo().isMac()) {
+        Path ideaPath = getToolPath().resolve("Contents").resolve("MacOS").resolve(IDEA);
```

Comment:
This looks odd to me. IMHO tool path should point to `software/intellij` if that contains `Contents/MacOS/idea` then already something went wrong before because our MacOS workaround did not work out.
It is sometimes helpful in such cases to first use `devon «tool»` on the according OS to manually test what is happening and what structure is created and then try to understand and make IDEasy creating the analog result (assuming that devonfw-ide is behaving correct, what is not always the case, as I faced today in some tests).

Reply:
So I just opened my MacBookPro and had a look for you:
```
➜  software ls -la intellij/
total 1872
drwxr-xr-x   4 hohwille  staff     128 Mar 19 09:15 .
drwxr-xr-x  14 hohwille  staff     448 Mar 12 06:03 ..
-rw-r--r--   1 hohwille  staff       9 Mar 19 09:15 .devon.software.version
-rwxr-xr-x   1 hohwille  staff  951888 Mar 12 06:03 idea
➜  software ls -la intellij
lrwxr-xr-x  1 hohwille  staff  50 Mar 19 09:15 intellij -> macos/intellij/IntelliJ IDEA CE.app/Contents/MacOS
```
So IMHO this is not yet correct and most probably will not work on MacOS unless I am missing something.